### ISSUE TITLE
Add aria-hidden=true to the icons in the toolbar

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/scholarsphere-3#612]
The font awesome icons in the toolbar should have aria-hidden=true for better accessibility.

![screen shot 2017-02-06 at 3 44 59 pm](https://cloud.githubusercontent.com/assets/4163828/22665585/43755652-ec83-11e6-931a-bcab827e314d.png)

**Comment by mtribone** [comment]
Also, we should reduce the top and bottom padding on the toolbar. It's HUGE...believe me...everybody who I talk to says that it is huge...and it's not just my head.

`.navbar-nav > li > a` in Sufia gets 15px and `.nav > li > a` in Sufia gets 10px. So like 25px of padding top and bottom.
